Mathlib/Algebra/Order/Floor.lean

```diff
@@ -435,6 +435,20 @@ theorem ceil_add_le (a b : α) : ⌈a + b⌉₊ ≤ ⌈a⌉₊ + ⌈b⌉₊ := b
   rw [ceil_le, Nat.cast_add]
   gcongr <;> apply le_ceil
 
+variable [Sub α] [OrderedSub α] [ExistsAddOfLE α]
+
+@[simp] lemma ceil_sub_natCast (a : α) (n : ℕ) : ⌈a - n⌉₊ = ⌈a⌉₊ - n := by
```

Comment:
Do we already have the `Int.floor` versions of these?

Reply:
Yes, but it's misnamed: `int.ceil_sub_nat`

Reply:
I'll make a deprecation PR

Reply:
#23563